.github/workflows/test_cra.yml

```diff
@@ -11,20 +12,50 @@ jobs:
       pull-requests: write
       contents: write
     name: Run code review agent on every pull request, respond to user comments
+
     steps:
+      - name: Set Options Environment Variable
+        run: |
+          echo "EVENT_NAME=${{ github.event_name }}" >> $GITHUB_ENV
+          GIT_DOMAIN_OPTION=""
+          if [[ -n "${{ vars.GIT_DOMAIN }}" ]]; then
+            GIT_DOMAIN_OPTION=" --git.domain=${{ vars.GIT_DOMAIN }}"
+          fi
+          STATIC_ANALYSIS_TOOL_OPTION=""
+          if [[ -n "${{ vars.STATIC_ANALYSIS_TOOL }}" ]]; then
+            STATIC_ANALYSIS_TOOL_OPTION=" --static_analysis_tool=${{ vars.STATIC_ANALYSIS_TOOL }}"
+          fi
+          REVIEW_SCOPE_OPTION=""
+          if [[ -n "${{ vars.REVIEW_SCOPE }}" ]]; then
+            REVIEW_SCOPE_OPTION=" --review_scope=${{ vars.REVIEW_SCOPE }}"
+          fi
+          EXCLUDE_BRANCHES_OPTION=""
+          if [[ -n "${{ vars.EXCLUDE_BRANCHES }}" ]]; then
+            EXCLUDE_BRANCHES_OPTION=" --exclude_branches=${{ vars.EXCLUDE_BRANCHES }}"
+          fi
+          EXCLUDE_FILES_OPTION=""
+          if [[ -n "${{ vars.EXCLUDE_FILES }}" ]]; then
+            EXCLUDE_FILES_OPTION=" --exclude_files=${{ vars.EXCLUDE_FILES }}"
+          fi
+          EXCLUDE_DRAFT_PR_OPTION=""
+          if [[ -n "${{ vars.EXCLUDE_DRAFT_PR }}" ]]; then
+            EXCLUDE_DRAFT_PR_OPTION=" --exclude_draft_pr=${{ vars.EXCLUDE_DRAFT_PR }}"
+          fi
+          echo "OPTIONS=--static_analysis.fb_infer.enabled=True --code_feedback=True --dependency_check.enabled=False --bee.path=/automation-platform  --bee.actn_dir=/automation-platform/default_bito_ad/bito_modules --git.access_token=${{ secrets.GIT_ACCESS_TOKEN }} --bito_cli.bito.access_key=${{ secrets.BITO_ACCESS_KEY }}${GIT_DOMAIN_OPTION}${STATIC_ANALYSIS_TOOL_OPTION}${REVIEW_SCOPE_OPTION}${EXCLUDE_BRANCHES_OPTION}${EXCLUDE_FILES_OPTION}${EXCLUDE_DRAFT_PR_OPTION}" >> $GITHUB_ENV
+
       - name: Code Review Agent - Issue Comment
-        id: cra_issues_comment
-        if: ${{ github.event_name == 'issue_comment' }}
-        uses: gitbito/codereviewagent@main
+        if: github.event_name == 'issue_comment'
+        uses: gitbito/codereviewagent@release/1.3.0
         with:
           pr: ${{ github.event.issue.pull_request.html_url }}
           command: ${{ github.event.comment.body }}
-          options: --static_analysis.fb_infer.enabled=True --code_feedback=True --dependency_check.enabled=False --bee.path=/automation-platform  --bee.actn_dir=/automation-platform/default_bito_ad/bito_modules --git.access_token=${{ secrets.GIT_ACCESS_TOKEN }} --bito_cli.bito.access_key=${{ secrets.BITO_ACCESS_KEY }}
+          options: ${{ env.OPTIONS }}
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/11#issuecomment-2244975494>#965150</a><span> - 07/23/2024, 11:31 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The dynamic environment variable '${{ env.OPTIONS }}' is used without ensuring it is properly set. If the variable is not set or is empty, it could lead to unexpected behavior or failures in the workflow.
</div>
<div id="fix">
<b>Fix:</b> Add a validation step to ensure that the '${{ env.OPTIONS }}' variable is set and not empty before using it in the workflow steps.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -44,6 +44,10 @@
        echo "OPTIONS=--static_analysis.fb_infer.enabled=True --code_feedback=True --dependency_check.enabled=False --bee.path=/automation-platform  --bee.actn_dir=/automation-platform/default_bito_ad/bito_modules --git.access_token=${{ secrets.GIT_ACCESS_TOKEN }} --bito_cli.bito.access_key=${{ secrets.BITO_ACCESS_KEY }}${GIT_DOMAIN_OPTION}${STATIC_ANALYSIS_TOOL_OPTION}${REVIEW_SCOPE_OPTION}${EXCLUDE_BRANCHES_OPTION}${EXCLUDE_FILES_OPTION}${EXCLUDE_DRAFT_PR_OPTION}" >> $GITHUB_ENV
 
    - name: Code Review Agent - Issue Comment
      if: github.event_name == 'issue_comment'
      uses: gitbito/codereviewagent@release/1.3.0
      with:
        pr: ${{ github.event.issue.pull_request.html_url }}
        command: ${{ github.event.comment.body }}
 +      options: ${{ env.OPTIONS }}
 +
 +  - name: Validate OPTIONS
 +    run: |
 +      if [ -z "${{ env.OPTIONS }}" ]; then
 +        echo "Error: OPTIONS environment variable is not set or is empty."
 +        exit 1
 +      fi
 
    - name: Code Review Agent action step
      if: github.event_name == 'pull_request'
      uses: gitbito/codereviewagent@release/1.3.0
      with:
        pr: ${{ github.event.pull_request.html_url }}
        command: review
        options: ${{ env.OPTIONS }}
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged